<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWidgetStatistical</class>
 <widget class="QWidget" name="CWidgetStatistical">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>903</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>统计与分析</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,12">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>高级检索</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="m_grpSurgDate">
         <property name="title">
          <string>手术日期</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>3</number>
            </property>
            <item>
             <widget class="QRadioButton" name="m_radThisWeek">
              <property name="text">
               <string>本周</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_radLastWeek">
              <property name="text">
               <string>上周</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_radThisMonth">
              <property name="text">
               <string>本月</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_radLastMonth">
              <property name="text">
               <string>上月</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_radThisYear">
              <property name="text">
               <string>本年</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_radLastYear">
              <property name="text">
               <string>去年</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_radFindExact">
              <property name="text">
               <string>精确</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_radFindRange">
              <property name="text">
               <string>范围</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,0,1">
            <item>
             <widget class="QDateEdit" name="m_dtStart">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="m_lblDate">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="m_dtEnd">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_grpQueryCondition">
         <property name="title">
          <string>检索条件</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QScrollArea" name="m_scrCondition">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <widget class="QWidget" name="m_wgtCondition">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>403</width>
               <height>712</height>
              </rect>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QVBoxLayout" name="m_layoutCondition"/>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>100000</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QPushButton" name="m_btnAddCondition">
           <property name="text">
            <string>添加条件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnResetCondition">
           <property name="text">
            <string>重置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnQuery">
           <property name="text">
            <string>检索</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,1">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>查询结果</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableView" name="m_tblvQueryAns"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,0,1">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>总计</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_lineSum"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>手术信息</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeView" name="m_treevSurgInfo"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
